scripts/mock-api.mjs

```diff
@@ -161,6 +248,49 @@ const getMarketRate = async (address) => {
 
 const getRoi = () => get(`${baseUrl}/roi`);
 
+const getHistoryPrices = async (baseToken, quoteToken, start, end) => {
+  const startTimestamp = convertToUnix(start);
+  const endTimestamp = convertToUnix(end);
+  const url = new URL(`${baseUrl}/history/prices`);
+  url.searchParams.set('baseToken', baseToken);
+  url.searchParams.set('quoteToken', quoteToken);
+  url.searchParams.set('start', startTimestamp);
+  url.searchParams.set('end', endTimestamp);
+  return get(url);
+};
+
+const getSimulatorData = async (
```

Comment:
If it were in the code (not a utils script) I would have asked you to use an object params and loop over it to set the searchParams.
But here it's ok I think

Reply:
Agree - still made the changes because it makes the code clearer anyways